src/Npgsql.GeoJSON/Npgsql.GeoJSON.csproj

```diff
@@ -7,7 +7,7 @@
     <Copyright>Copyright 2018 © The Npgsql Development Team</Copyright>
     <Company>Npgsql</Company>
     <PackageTags>npgsql postgresql postgres postgis geojson spatial ado ado.net database sql</PackageTags>
-    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
```

Comment:
As I wrote elsewhere we may be able to remove .NET Framework as a target of the plugins, since they don't actually need them. We just need to test whether a .NET Framework application referencing npgsql and the plugin properly gets the .NET Framework version of npgsql and the .NET Standard version of the plugin.

But this can be handled in a separate PR.

Reply:
Since these changes already required so many modifications due to `#if` blocks throughout the solution, I figured it would be best to just move everything forward before removing the framework altogether. 

Once this is merged, I'll circle back to test/submit that as a separate PR.

Reply:
Makes sense.